src/Symfony/Component/Validator/Constraints/NullValidator.php

```diff
@@ -11,25 +11,12 @@
 
 namespace Symfony\Component\Validator\Constraints;
 
-use Symfony\Component\Validator\Constraint;
-use Symfony\Component\Validator\ConstraintValidator;
-
 /**
  * @author Bernhard Schussek <bschussek@gmail.com>
  *
  * @api
+ *
+ * @deprecated Deprecated since version 2.3.28, to be removed in 3.0. Use
```

Comment:
So trigger_errors in 2.7 only?

This discussion had appeared earlier in this thread as well, I think back then it was decided to make an exception? Though @fabpot did mention the deprecations would only be necessary if we wanted to make the branch PHP7 compatible.